framework/src/controller/controller.js

```diff
@@ -147,15 +151,16 @@ class Controller {
 
 	async _loadModules(modules) {
 		// To perform operations in sequence and not using bluebird
-
 		// eslint-disable-next-line no-restricted-syntax
 		for (const alias of Object.keys(modules)) {
-			// eslint-disable-next-line no-await-in-loop
-			await this._loadInMemoryModule(
-				alias,
-				modules[alias].klass,
-				modules[alias].options
-			);
+			const { klass, options } = modules[alias];
```

Comment:
What does `klass` mean here?

Reply:
`klass` is commonly used to name a variable that holds a class object and here it holds the module class the will be instantiated when loading the modules